ern-util/src/fileUtil.js

```diff
@@ -35,3 +38,15 @@ export async function writeFile (
     })
   })
 }
+
+/**
+ * Recursively apply file mode for a given path
+ * @param fileMode
+ * @param path
+ * @returns {Promise.<void>}
+ */
+export async function chmodr (fileMode: string, path: string) {
```

Comment:
Could you please use `shelljs` library to `chmod` instead of running the command.
Otherwise running the platform directly is not cross platform and will fail on Windows.
https://github.com/shelljs/shelljs#chmodoptions-symbolic_mode-file
Make sure to import `shell` from `ern-util`. Do not import `shell` library directly

Reply:
This function is not `async`, it does not return a `Promise`. 
`exec` returns a `ChildProcess` instance, not a `Promise`.
Would suggest just keeping it sync (after refactoring with `shelljs`) instead. `chmoding` is pretty quick anyway.